lib/cli/src/js-package-manager/NPMProxy.ts

```diff
@@ -15,8 +18,18 @@ export class NPMProxy extends JsPackageManager {
     return `npm run ${command}`;
   }
 
+  getInstallArgs(): string[] {
+    if (!this.installArgs) {
+      const version = this.executeCommand('npm', ['--version']);
```

Comment:
Wondering if you need some `.trim()` here?

Reply:
tested by hand -- doesn't seem to matter to semver, but good thought